R/plotreg.R

```diff
@@ -1,394 +1,462 @@
-        pch = 20, cex = cex)
-  }
-  if (any(!signif)) {
-    points(estimates[!signif], num + 1 - which(!signif), col = insignif.dark, 
-        pch = 20, cex = cex)
-  }
-}
-
-
-# plotreg function
-plotreg <- function(l, file = NULL, custom.model.names = NULL, 
-    custom.coef.names = NULL, custom.note = NULL, override.coef = 0, 
-    override.se = 0, override.pval = 0, override.ci.low = 0, 
-    override.ci.up = 0, omit.coef = NULL, reorder.coef = NULL, 
-    ci.level = 0.95, use.se = FALSE, mfrow = TRUE, xlim = NULL, 
-    cex = 2.5, lwd.zerobar = 4, lwd.vbars = 1, lwd.inner = 7, 
-    lwd.outer = 5, ylab.cex = 1.0, signif.light = "#fbc9b9", 
-    signif.medium = "#f7523a", signif.dark = "#bd0017", 
-    insignif.light = "#c5dbe9", insignif.medium = "#5a9ecc", 
-    insignif.dark = "#1c5ba6", ...) {
-  
-  if (!is.null(omit.coef) && !is.na(omit.coef) && !is.character(omit.coef)) {
-    stop("omit.coef must be a character string!")
-  }
-  if (is.null(omit.coef)) {
-    omit.coef <- NA
-  }
-  
-  if (length(use.se) == 1) {
-    use.se <- rep(use.se, length(l))
-  }
-  
-  # extract texreg objects and override data
-  models <- get.data(l, ...)
-  models <- override(models, override.coef, override.se, override.pval, 
-      override.ci.low, override.ci.up)
-  
-  # custom model names
-  model.names <- character()
-  if (is.null(custom.model.names)) {
-    model.names <- paste("Model", 1:length(l))
-  } else if (length(custom.model.names) == 1) {
-    model.names <- rep(custom.model.names, length(l))
-  } else if (length(custom.model.names) != length(l)) {
-    stop(paste("The 'custom.model.names' argument must have the same length as",
-        "the 'l' argument."))
-  } else {
-    model.names <- custom.model.names
-  }
-  
-  # file output; check file extension
-  if (!is.null(file) && !is.na(file)) {
-    if (grepl(".pdf$", file)) {
-      pdf(file, ...)
-    } else if (grepl(".jpe*g$", file)) {
-      jpeg(file, ...)
-    } else if (grepl(".png$", file)) {
-      png(file, ...)
-    } else if (grepl(".bmp$", file)) {
-      bmp(file, ...)
-    } else if (grepl(".tiff*$", file)) {
-      tiff(file, ...)
-    } else if (grepl(".ps$", file)) {
-      postscript(file, ...)
+    
+    # extract texreg objects and override data
+    models <- get.data(l, ...)
+    models <- override(models, override.coef, override.se, override.pval, 
+                       override.ci.low, override.ci.up)
+    # custom model names
+    model.names <- character()
+    if (is.null(custom.model.names)) {
+        model.names <- paste("Model", 1:length(l))
+    } else if (length(custom.model.names) == 1) {
+        model.names <- rep(custom.model.names, length(l))
+    } else if (length(custom.model.names) != length(l)) {
+        stop(paste("The 'custom.model.names' argument must have the same length as", "the 'l' argument."))
     } else {
-      stop("File extension not recognized.")
+        model.names <- custom.model.names
```

Comment:
Proper indentation is two spaces. Maybe you can automatically let RStudio correct the indentation of your function by selecting all lines and pressing Ctrl-I in RStudio... I think there is also a setting in the RStudio options where you can ensure that two spaces represent a single indentation level.